ragna/deploy/_ui/components/file_uploader.py

```diff
@@ -115,15 +115,20 @@ def perform_upload(self, event=None, after_upload_callback=None):
                 .fileListContainer {
                     display:flex;
                     flex-direction: row;
-                    height: 44px;
-                    overflow:scroll;
-                    padding-top:14px;
+                    flex-wrap: wrap;
```

Comment:
Introduces wrapping so that if the pills are wider than the modal, they 'wrap' around to the next line, and scroll along the Y-axis. I have removed the scroll on the X-axis as I found having too scrolls to be hard to follow. I have also added more spacing to compensate for the fact that the file upload container no longer shrinks when a pill is added (see previous comment).